requirements/torch_windows.txt

```diff
@@ -1,7 +1,5 @@
 #pip install torch torchvision torchaudio
-wheel
 torch==2.0.0; sys_platform == 'win32'
```

Comment:
I don't think sys_platform does anything here

Reply:
right.. lol. I dont know why this has always been there

Reply:
I left it as is